File: 3701-compute-alternating-sum/3701-compute-alternating-sum.cpp
class Solution {
public:
    int alternatingSum(vector<int>& nums) {
        int s=nums[0];
        for(int i=1;i<nums.size();i++){
            if(i%2==0 ){
                s+=nums[i];
            }
            else{
                s-=nums[i];
            }
        }
        return s;
    }
};

File: 3701-compute-alternating-sum/3701-compute-alternating-sum.py
class Solution(object):
    def alternatingSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        se=0
        so=0
        for i in range(0,len(nums)):
            if(i%2!=0):
                so+=nums[i]
            else:
                se+=nums[i]
        return se-so
        